.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,37 @@
+---
+labels: bug
+name: "Bug Report"
+about: "Report an error condition or unexpected behavior"
+---
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+<!-- delete any unused steps -->
+1.
+2.
+3.
+4.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+```[tasklist]
```

Comment:
why a "tasklist" ?

Reply:
The reason for using a tasklist here is to allow GitHub to know about the issue dependencies/hierarchies so that information can be reflected in the project board.